docs/basic/dictionary.md

```diff
@@ -352,7 +352,7 @@ profile = {
 }
 
 print(list(profile.keys()))
-# output ➜ ['id', 'name', 'is_female']
+# output ➜ ['id', 'name', 'hobbies', 'is_female']
```

Comment:
I updated the output of some of the codes under this section instead of the changing the data, because the code here should match with anything on https://github.com/novalagung/dasarpemrogramanpython-example